applications/GeoMechanicsApplication/custom_elements/small_strain_U_Pw_diff_order_element.cpp

```diff
@@ -1837,6 +1836,21 @@ std::vector<double> SmallStrainUPwDiffOrderElement::CalculateRelativePermeabilit
     return result;
 }
 
+std::vector<double> SmallStrainUPwDiffOrderElement::CalculateBishopCoefficients(const std::vector<double>& rFluidPressures) const
+{
```

Comment:
This function is a copy of the correspondent function in U_Pw_small_strain. Does it mean that we accept it now having in mind to remove DiffOrder or U_Pw later?

Reply:
Yes indeed, I would prefer not having to duplicate it, but since we're looking to merge them, it seems acceptable for now.